pkg/logger/operationid_middleware.go

```diff
@@ -49,3 +55,7 @@ func GetOperationID(ctx context.Context) string {
 	}
 	return ""
 }
+
+func operationIDAttribute(id string) attribute.KeyValue {
+	return attribute.String(strings.ToLower(string(OpIDKey)), id)
```

Comment:
(nit) should we define our own constant for the attribute name?

Reply:
The `opID` constant already exists so we can just reuse it. The only diff is it is `opID` instead of `opid` hence the `ToLower`. I didn't want to create another constant that is basically the same.